translations/tiled_pt.ts

```diff
@@ -2488,12 +2488,12 @@ Linha %1, coluna %2</translation>
     <message>
         <location line="+250"/>
         <source>Tileset Columns Changed</source>
-        <translation type="unfinished"></translation>
+        <translation>Colunas do Tileset Alteradas</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The number of tile columns in the tileset &apos;%1&apos; appears to have changed from %2 to %3. Do you want to adjust tile references?</source>
-        <translation type="unfinished"></translation>
+        <translation>O número de colunas do tile no tileset &apos;%1&apos; parece ter mudado de%2 para %3. Você deseja ajustar as referências do tile?</translation>
```

Comment:
I assume there should be a space before the `%2` here, which I'll add before pushing the change.